client/post-editor/editor-sharing/accordion.jsx

```diff
@@ -46,12 +43,15 @@ const EditorSharingAccordion = React.createClass( {
 		}
 
 		const skipped = PostMetadata.publicizeSkipped( post );
```

Comment:
Though I suppose it might change the behavior such that if we know connections before we know the post, it would show all connection labels in the subtitle since it would consider none of them to have been skipped. If we want to preserve current behavior, probably best to leave the condition as-is.